rs/node_rewards/proto/src/lib.rs

```diff
@@ -0,0 +1,155 @@
+use crate::pb::v1::{
+    NodeMetricsDailyStored, SubnetIdKey, SubnetMetricsDailyKeyStored, SubnetMetricsDailyValueStored,
+};
+use candid::Principal;
+use ic_base_types::{NodeId, PrincipalId, SubnetId};
+use ic_stable_structures::storable::Bound;
+use ic_stable_structures::Storable;
+use prost::Message;
+use rewards_calculation::types::{NodeMetricsDailyRaw, SubnetMetricsDailyKey};
+use std::borrow::Cow;
+
+pub mod pb;
+
+// Maximum sizes for the storable types chosen as result of test `max_bound_size`
+const MAX_BYTES_SUBNET_ID_STORED: u32 = 33;
+const MAX_BYTES_NODE_METRICS_STORED_KEY: u32 = 44;
+const PRINCIPAL_MAX_LENGTH_IN_BYTES: usize = 29;
+
+pub const MIN_PRINCIPAL_ID: PrincipalId = PrincipalId(Principal::from_slice(&[]));
+pub const MAX_PRINCIPAL_ID: PrincipalId = PrincipalId(Principal::from_slice(
+    &[0xFF_u8; PRINCIPAL_MAX_LENGTH_IN_BYTES],
+));
+
+#[test]
+fn max_bound_size() {
```

Comment:
You should put all the tests in `mod tests` at the bottom of the file and only build them with cfg(test)